frontend/src/components/RequireClaimsButtons.vue

```diff
@@ -1,48 +1,79 @@
 <template>
   <span>
-    <v-btn :key="button.text" v-for="button in buttons" v-bind="button.props">
+    <v-btn :key="`${button.text}-${index}`" v-for="(button, index) in routeButtons" v-bind="button.props">
       {{ button.text }}
     </v-btn>
   </span>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
-import firebase from "firebase/app";
-import "firebase/auth";
+import _ from "lodash";
+import { mapActions } from "vuex";
+import { RouteConfig } from "vue-router";
+import { filterRoutesByClaims, hasClaim } from "@/router";
 
 @Component({
-  name: "RequireClaimsButtons"
+  name: "RequireClaimsButtons",
```

Comment:
Please rename this component.
I'd suggest `HomePageButtons`, as this component is going to be used only there. The fact that the buttons are shown according to the claims is secondary. The primary is the purpose of the component—shortcuts on the home page.